### DISCUSSION TITLE
What code theme are you using on your website?

### DISCUSSION THREAD

**Discussion by Kotletka89** [floating-ui/floating-ui#1913] [General]
Is it available on vscode?

**Comment by atomiks** [comment]
https://github.com/atomiks/moonlight-vscode-theme

**Reply by Kotletka89** [reply]
Ohhh, actually I'm was asking for a theme that used on popper js website. Thanks, but do you know a theme that used on popper js?

**Reply by atomiks** [reply]
Prism's `base2tone-pool-dark`, but was modified a bit, not sure if it's available for VS Code though.
